Tribler/community/allchannel/community.py

```diff
@@ -218,7 +217,7 @@ def create_channelcast(self):
                 # we've send something to this address, add to blocklist
                 self._blocklist[candidate] = now
 
-                nr_torrents = sum(len(torrent) for torrent in torrents.values())
+                nr_torrents = sum(len(infohash) for infohash in torrents.itervalues())
```

Comment:
Oh, made a mistake here. Should be `infohashes` instead. Thanks! @NielsZeilemaker